components/home/CountDown.vue

```diff
@@ -0,0 +1,197 @@
+<script setup lang="ts">
+  export interface CountDownContent {
+    futureDate: string
+    futureTime: string
+  }
+
+  const cdProps = defineProps({
+    content: { type: Object as PropType<CountDownContent>, required: true },
+  })
+
+  export interface TimeDiffReturn {
+    days: string
+    hours: string
+    minutes: string
+    seconds: string
+  }
+
+  const timeDiff = ref<object | null>(null)
+
+  onMounted(() => {
+    setInterval(() => {
+      timeDiff.value = calculateTimeDifference(
+        cdProps.content.futureDate,
+        cdProps.content.futureTime
+      )
+    }, 1000)
+  })
+</script>
+
+<template>
+  <!-- eslint-disable-next-line vue/max-len -->
+  <!-- Card containing 2 rows and a VCardAction at the bottom. First row is for event take off string, second row contains 4 cols with 2 rows each. The 2 rows in these 4 cols contains the number for days/hours/minutes/seconds and the corresponding text ("days/hours/minutes/seconds"), respectively -->
```

Comment:
I think you can omit the description comment here. If we have the time in the spring, we will write documentation for the code in form of a WIKI. Hence we can include a component description there.

Optionally, you can leave a comment within the `<script></script>` in the `CountDownContent` interface to specify the format the string format should be.

Reply:
Agreed